src/lib/models/keyframe.js

```diff
@@ -115,7 +121,20 @@ module.exports = class Keyframe {
     return [errors.length === 0, errors]
   }
 
-  schema() {
+  validateKeyframe(obj, isCore=false){
+      if (isCore){
+          return validateSchema(obj, this.coreSchema())
+      }
+      return validateSchema(obj, this.schema())
+  }
+
+  schema(){
+    // Full keyframe schema, including any customization or extension
+    return _.merge(this.coreSchema(), {})
```

Comment:
So this will eventually be extended when we know what additional schemas we want to support?

Reply:
yeah! The `{}` part can have any schema we like and will be merged with the core schema!